modules/installing-private-image-registry-private-azure.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+//* registry/configuring_registry_storage-azure.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-private-image-registry-private-azure"]
+= Optional: Preparing a private Microsoft Azure cluster for a private image registry
+
+By installing a private image registry on a private Microsoft Azure cluster, you can create private endpoints. Private endpoints disable public facing endpoints to the internal registry, adding an extra layer of security to your {product-title} deployment.
```

Comment:
Let's also use `private storage endpoints` everywhere on this PR instead of just `private endpoints` (so there's no chance of confusing them with route endpoints)
```suggestion
By installing a private image registry on a private Microsoft Azure cluster, you can create private storage endpoints. Private storage endpoints disable public facing endpoints to the registry's storage account, adding an extra layer of security to your {product-title} deployment.
```

Reply:
Fixed!